core/domain/suggestion_services.py

```diff
@@ -1070,7 +1072,8 @@ def get_reviewable_question_suggestions_by_offset(
     """
     suggestions, next_offset = (
         suggestion_models.GeneralSuggestionModel
-        .get_in_review_question_suggestions_by_offset(limit, offset, user_id))
+        .get_in_review_question_suggestions_by_offset(
+            limit, offset, user_id, sort_key))
```

Comment:
This should be `sort_key=sort_key` since it's optional in the function being called.